clang/lib/Driver/Driver.cpp

```diff
@@ -1221,9 +1221,11 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
   // We need to generate a SYCL toolchain if the user specified -fsycl.
   // If -fsycl is supplied without any of these we will assume SPIR-V.
   // Use of -fsycl-device-only overrides -fsycl.
+  // Use of -fsyclbin also enables SYCL offloading compialtion.
```

Comment:
> Use of -fsyclbin also enables SYCL offloading compialtion

Does this mean you can pass `-fsyclbin` without `-fsycl` to enable SYCL offloading?

Reply:
Use of `-fsyclbin` implies `-fsycl` but it isn't enabling offloading, but rather just the device compilation.  I will update the comment to be more clear here.